devices.js

```diff
@@ -2527,6 +2527,52 @@ const devices = [
             execute(device, actions, callback);
         },
     },
+
+    // Keen Home
+    {
+        zigbeeModel: ['SV01-410-MP-1.4'],
+        model: 'SV01-410-MP-1.4',
+        vendor: 'Keen Home Inc.',
+        description: 'Smart Vent',
+        supports: 'open, close, position, temperature, pressure',
+        fromZigbee: [
+            fz.brightness,
+            fz.state_change,
+            fz.state,
+            fz.brightness_report,
+            fz.generic_temperature,
+            fz.generic_pressure,
+            fz.generic_battery,
+        ],
+        toZigbee: [
+            tz.on_off,
+            tz.light_brightness,
+            tz.ignore_transition,
+        ],
+        configure: (ieeeAddr, shepherd, coordinator, callback) => {
+            const device = shepherd.find(ieeeAddr, 1);
+            const actions = [
+                (cb) => device.write('ssIasZone', 'iasCieAddr', coordinator.device.getIeeeAddr(), cb),
+                (cb) => device.functional('ssIasZone', 'enrollRsp', {enrollrspcode: 0, zoneid: 23}, cb),
```

Comment:
As mentioned, this is a smart vent, do I need these 2 lines above?

Reply:
Probably not.